src/app/static/src/app/store/translations/locales.ts

```diff
@@ -173,6 +175,53 @@ export interface Translations {
 }
 
 const en: Translations = {
+    accessibility: "Accessibility",
+    accessibilityContent: "<h1>Accessibility on NAOMI</h1>\n" +
```

Comment:
We don't capitalise 'Naomi' in the navbar so I guess we shouldn't do so here either.